values.yaml

```diff
@@ -37,7 +37,7 @@ config:
     # db:
     # user:
     # password:
-    # ssl_enabled:
+    # ssl_enabled: this is equivalent to the POSTGRES_SSL_ENABLED environment variable (whether to enable SSL for connecting to the Retool Postgres database). Place here instead of in the env block.
```

Comment:
```suggestion
    # ssl_enabled: whether to enable SSL for connecting to the Retool Postgres database, equivalent to the `POSTGRES_SSL_ENABLED` environment variable. Place here instead of in the env block.
```

Reply:
+1